src/sys.zig

```diff
@@ -562,9 +525,9 @@ pub fn openatWindows(dir: bun.FileDescriptor, path: []const u16, flags: bun.Mode
 ///
 /// In the zig standard library, messing up the input to their equivalent
 /// will trigger `unreachable`. Here there will be a debug log with the path.
-pub fn ntCreateFile(
+pub fn openFileAtWindowsNtPath(
     dir: bun.FileDescriptor,
-    path_maybe_leading_dot: []const u16,
+    path: [:0]const u16,
```

Comment:
why is this required to be a sentinel? it is converted to `windows.UNICODE_STRING`.

`[]const u16` can be downcasted from `[:0]const u16` without any casts.

Reply:
Good point, and this was already correct for the directory version of this function. updated.